cmd/build.go

```diff
@@ -157,7 +179,39 @@ func newBuildCmd() *cobra.Command {
 				return err
 			}
 
-			api := apiNew(entry.APIURL, entry.Token)
+			if entry.UA == "" {
+				fmt.Println("sd-local collects unique UUIDs for statistical surveys.")
+				fmt.Println("You can reset it later by removing the UA key from config.")
+				fmt.Print("Would you please cooperate with the survey? [y/N]: ")
+
+				input, err := bufio.NewReader(os.Stdin).ReadString('\n')
+				if err != nil {
+					return err
+				}
+				input = strings.TrimSuffix(input, "\n")
+				if input == "y" || input == "Y" || input == "yes" || input == "Yes" {
+
```

Comment:
sd-local should print some message after user input.

Reply:
What message do you want?

Reply:
something like `UA key has been added to <config file path>` / `UA key is not set`